<ui version="4.0" >
 <class>InstallProgressFrm</class>
 <widget class="QDialog" name="InstallProgressFrm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Progress</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="labelHeader" >
     <property name="text" >
      <string>Progress</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QListWidget" name="listProgress" >
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QProgressBar" name="progressBar" >
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" >
    <widget class="QPushButton" name="buttonAbort" >
     <property name="text" >
      <string>&amp;Abort</string>
     </property>
     <property name="icon" >
      <iconset resource="rbutilqt.qrc" >:/icons/process-stop.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rbutilqt.qrc" />
 </resources>
 <connections/>
</ui>
